### PR TITLE
tarssh: init at 0.4.0

### PR DESCRIPTION
**Pull request by Sohalt**
###### Motivation for this change

I want to use this.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/tarssh/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    owner = "Freaky";
+    repo = pname;
+    sha256 = "0fm0rwknhm39nhd6g0pnxby34i5gpmi5ri795d9ylsw0pqwz6kd0";
+  };
+
+  cargoSha256 = "108xdpgfgfd4z455snif0mzbla0rv8gjqxci5qgwjzfyshwkprgv";
+
+  meta = with stdenv.lib; {
+    description = "A simple SSH tarpit inspired by endlessh";
+    homepage = "https://github.com/databricks/click";
```
Comment (ajs124) [inline pkgs/servers/tarssh/default.nix:20]: Are you sure that's the right homepage?

Comment (Sohalt) [reply pkgs/servers/tarssh/default.nix]: good catch

**Review by omasanori — APPROVED** [review]

**Comment by omasanori** [comment]
Built and tested on NixOS 20.09 x86_64, with or without `-v` and `-l` options, listen on IPv4 and IPv6. WFM.

**Comment by AndersonTorres** [comment]
Best motivation ever!

**Pull request merged**